iree-prof-tools/graph.cc

```diff
@@ -0,0 +1,101 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree-prof-tools/graph.h"
+
+#include "third_party/llvm-project/llvm/include/llvm/Support/JSON.h"
```

Comment:
nit: included in graph.h.

Reply:
Include-What-You-Use rule requires .cc to include files again since .h file might be changed to not include it.